### DISCUSSION TITLE
module scripts installing in the wrong folder?

### DISCUSSION THREAD

**Discussion by wcrisafu** [oqtane/oqtane.framework#717] [General] [answered]
Hi,

I am trill testing and trying to get familiar with Oqtane as I have not developed in many years. I have a Windows 10 PC with IIS and SQL Srv for developer. I compiled the DNF.Project, Oqtane.Blogs and then the Oqtane framework for release (all of them out of github, no modifications). In IIS I created a new site (C:\inetpub\wwwroot\Oqtane) and published to the same folder. Because i have problems with IIS (still playing with access rights and application pool), I decided to use powershell instead. I started powershell as administrator, cd to the Oqtane folder and started Oqtane with "dotnet oqtane.server.dll".

Installation went fine but the scripts folders for the modules were created one level higher "C:\inetpub\Oqtane\wwwroot\Modules" (the dlls went to the correct folder). Does anyone know why?

**Comment by sbwalker** [comment, marked as answer]
Did you follow the steps in this blog: https://www.oqtane.org/Resources/Blog/PostId/542/installing-oqtane-on-iis

**Reply by wcrisafu** [reply]
Yes and No. Following the steps up to a point. 
-I installed and then uninstalled the "Url Rewrite Module"as I was not sure it was creating problems to my PC (need more investigation).
-I did not use the release zip file as I keep downloading the latest updated code from github, recompile the code + modules and publish.
-I keep getting the installation screen (even after I just finished installing and configuring via Powershell) and after i put the DB details I get an "Installation not Authorized".

**Reply by wcrisafu** [reply]
I deleted everything and followed the blog step by step (except for the URL Rewrite) and After a few more bumps (the ID used for the DB creation "NT AUTHORITY\SYSTEM" did not have creation rights, so it kept failing), i successfully installed and configured a copy of the code running on IIS :)

Do I need the URL Rewrite? Should I put more time on it?

**Reply by sbwalker** [reply]
What database engine are you using - SQL Server or LocalDB? The blog has instructions for LocalDB and includes information on the necessary permission settings.

**Reply by wcrisafu** [reply]
I installed the SQL developer edition. It has more features, in case I want to do more advanced stuff :)
